bin/game.rb

```diff
@@ -14,18 +16,23 @@ def initialize
     @game_ui = UI.new(stdout: STDOUT, stdin: STDIN)
     game = Game.new(board_width: 3)
     @board_gateway = InMemoryBoardGateway.new(game.board)
+    @board_gateway_hack = InMemoryBoardGateway.new(game)
```

Comment:
This is very suspicious looking...

Reply:
agree with Yusuf here

Reply:
When you forget to remove your hacks before pushing to github, the game/UI is incomplete... 😅